cmd/github/github.go

```diff
@@ -5951,7 +5963,7 @@ func (j *DSGitHub) GetModelDataPullRequest(ctx *shared.Ctx, docs []interface{})
 		updatedOn := j.ItemUpdatedOn(doc)
 		githubRepoName, _ := doc["github_repo"].(string)
 		repoShortName, _ := doc["repo_short_name"].(string)
-		repoID, err = repository.GenerateRepositoryID(githubRepoName, j.URL, source)
+		repoID, err = repository.GenerateRepositoryID(j.SourceID, j.URL, source)
```

Comment:
what if somebody didn't pass `source-id` ? it will send empty - I think you should check for empty and use `githubRepoName` if `j.SourceID` is not set.
Is the same code going to be added to all connectors? I mean what is wrong with the current code? `githubRepoName` contains repo name - it is correct according to `lfx-event-schema`'s description on how to use `repository.GenerateRepositoryID` function - not sure what is wrong with this?

Reply:
Fayaz suggested that for github, we use the repository id from the github api as the source-id because its unique for each repository. This will always be passed to the connector by the onboarding api. I'll add a check for empty ones just to make sure it always has a value.

Reply:
sounds good.